projects/element-ng/copyright-notice/si-copyright-notice.component.html

```diff
@@ -1 +1,5 @@
-<p>© {{ copyrightInfo() }}</p>
+<p>
+  ©
+  <span class="company">{{ company() }}</span>
```

Comment:
This is not a good idea, as copyrights follow different formatting. For example, check the reuse tool and you find other best practises. The copyright should be provided as a string. This brings most flexibility.